jobs/build/build-sync/Jenkinsfile

```diff
@@ -68,30 +68,32 @@ node {
     }
 
     try {
-	// This stage is safe to run concurrently. Each build runs
-	// these steps in its own directory.
-        stage("Generate inputs") { build.buildSyncGenInputs() }
-	// Allow this job to run concurrently for different
-	// versions. That is to say, do not allow builds for the same
-	// version to run the business logic concurrently.
-        lock("mirroring-lock-OCP-${params.BUILD_VERSION}") {
-            stage("oc image mirror") { build.buildSyncMirrorImages() }
-	}
-        lock("oc-applying-lock-OCP-${params.BUILD_VERSION}") {
-            stage("oc apply") { build.buildSyncApplyImageStreams() }
+        sshagent(["openshift-bot"]) {
```

Comment:
I don't see that anything uses github ssh in here, so I'm going to remove this to prevent confusion.